src/module.d.ts

```diff
@@ -308,14 +308,22 @@ export namespace plugins {
     interface PositionSettings {
         // The height of the table
         tableHeight?: number|string;
+
         // The width of the table
         tableWidth?: number|string;
+
         // The minimum row height
         rowHeight?: number|string;
+
+        // TODO: Unused?
```

Comment:
@joellanciaux I'll look into this if you don't know offhand but are we using this still?